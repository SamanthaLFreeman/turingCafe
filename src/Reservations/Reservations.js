import React from 'react';

const Reservations = ({reservations}) => {
  return (
    <section>

    </section>
  )
}

export default Reservations;