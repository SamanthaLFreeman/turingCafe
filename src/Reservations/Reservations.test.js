import React from 'react';
import { shallow } from 'enzyme';
import Reservations from './Reservations';

describe('Reservations', () => {
  let wrapper;
  const reservationsMock = [{
    name: 'Sam',
    date: '1/14',
    time: '16:00',
    number: 2,
    id: 1
  }]

  beforeEach(() => {
    wrapper = shallow(
      <Reservations
        reservations={reservationsMock} />)
  })

  it('should match the snapshot with all of the data passed through', () => {
    expect(wrapper).toMatchSnapshot();
  })
})