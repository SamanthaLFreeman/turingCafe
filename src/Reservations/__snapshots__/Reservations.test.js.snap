// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Reservations should match the snapshot with all of the data passed through 1`] = `
<section>
  <Reservation
    key="1"
    reservation={
      Object {
        "date": "1/14",
        "id": 1,
        "name": "Sam",
        "number": 2,
        "time": "16:00",
      }
    }
  />
</section>
`;
